ares/n64/cpu/recompiler.cpp

```diff
@@ -32,6 +32,7 @@ auto CPU::Recompiler::emit(u32 vaddr, u32 address, bool singleInstruction) -> Bl
 
   auto block = (Block*)allocator.acquire(sizeof(Block));
   beginFunction(3);
+  emitZeroClear(0);
```

Comment:
Is this needed? Shouldn't this always be zero on entry?

Reply:
You're right. I had missed the `for(auto& r : ipu.r) r.u64 = 0;` line in `CPU::power`.